Descent3/LoadLevel.cpp

```diff
@@ -3588,6 +3588,86 @@ static inline char* GetCurrentSumString() {
   return output_buf;
 }
 
+bool GetLevelInfo(const std::filesystem::path &filename, level_info &info) {
+  CFILE *ifile = cfopen(filename, "rb");
+  bool found = false;
```

Comment:
variable is only used much later, can be declared before the `while` loop below

Reply:
There try-catch block, we cannot declare variable inside it.